rotary_utils/rotary_utils.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+
+"""
+Description: A command-line interface for the Rotary-utils toolkit.
+Copyright: Jackson M. Tsuji and Lee H. Bergstrand 2024
+"""
+
+import logging
+import argparse
+
+import rotary_utils.repair as repair
+
+# Initialize the root logger with a stream handler
+logger = logging.getLogger()
+formatter = logging.Formatter('[ %(asctime)s ]: %(levelname)s: %(filename)s: %(funcName)s: %(message)s',
+                              datefmt='%Y-%m-%d %H:%M:%S')
+stream_handler = logging.StreamHandler()
+stream_handler.setFormatter(formatter)
+logger.addHandler(stream_handler)
+
+
+def main():
+    """
+    Collects input arguments and selects a command to perform.
+    """
+
+    """
+    When installing through pip with pyprojects.toml a new python script is generated
+    that calls main() out of rotary_utils.py. Run parser inside main() so it can be called 
+    externally as a function.
+    """
+    parser = parse_cli()
+    args = parser.parse_args()
+
+    # Select the sub-command to run.
+    if hasattr(args, 'repair'):
+        repair.main(args)
```

Comment:
Would it be better to move the `main` function of `repair.py` to the `rotary_utils.py` file? Not sure of best practices here. I ask a similar question above about the `parse_cli` function of `repair.py`. Thanks!

Reply:
I would move as much of the code as can be shared across commands to `rotary_utils.py` or to functions called out of a utility library by  `rotary_utils.py` .

Reply:
Makes sense -- thanks! Let's see if I can centralize some of the code, e.g., for handling logging files.

Reply:
Several steps were taken to centralize code in #2

Reply:
Great!